query.go

```diff
@@ -574,3 +577,132 @@ func aurInfoPrint(names []string) ([]*rpc.Pkg, error) {
 
 	return info, nil
 }
+
+func aurPkgbuilds(names []string) ([]string, error) {
+	pkgbuilds := make([]string, 0, len(names))
+	var mux sync.Mutex
+	var wg sync.WaitGroup
+	var errs multierror.MultiError
+
+	makeRequest := func(name string) {
+		defer wg.Done()
+
+		values := url.Values{}
+		values.Set("h", name)
+
+		url := "https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?"
+
+		resp, err := http.Get(url + values.Encode())
+		if err != nil {
+			errs.Add(err)
+			return
+		}
+		defer resp.Body.Close()
+
+		body, readErr := ioutil.ReadAll(resp.Body)
+		pkgbuild := string(body)
+
+		if readErr != nil {
+			errs.Add(readErr)
+			return
+		}
+
+		if strings.Contains(pkgbuild,
+			"<div class='content'><div class='error'>Invalid branch: "+name+"</div>") {
```

Comment:
`https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=ewemwe` seems to return 404, why not check `resp.StatusCode` instead, and add err after `http.Get` if `StatusCode != 200`